resources/setup-spin

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+# This script uses sh instead of bash to be compatible with as many distros as possible.
+
```

Comment:
Should we have `errexit` etc set? (Well, since this is `sh`, `set -eu` I guess…)